src/arch/wasm/CodeGen.zig

```diff
@@ -3338,9 +3339,14 @@ fn airCmpVector(func: *CodeGen, inst: Air.Inst.Index) InnerError!void {
 fn airCmpLtErrorsLen(func: *CodeGen, inst: Air.Inst.Index) InnerError!void {
     const un_op = func.air.instructions.items(.data)[inst].un_op;
     const operand = try func.resolveInst(un_op);
+    const sym_index = try func.bin_file.getGlobalSymbol("__zig_lt_errors_len", null);
+    const errors_len = WValue{ .memory = sym_index };
 
-    _ = operand;
-    return func.fail("TODO implement airCmpLtErrorsLen for wasm", .{});
+    try func.emitWValue(operand);
+    const errors_len_val = try func.load(errors_len, Type.err_int, 0);
+    const result = try func.cmp(.stack, errors_len_val, Type.err_int, .lte);
```

Comment:
```suggestion
    const result = try func.cmp(.stack, errors_len_val, Type.err_int, .lt);
```

Reply:
Perhaps it's a misunderstanding from me. But error values start at `1` rather than `0`, right? That means if the global error set contains a single error, this would need to pass for the value `1`. This was the reasoning for me as to why I changed to `lte` from `lt` during testing.

Reply:
Error values that indicate errors indeed start at 1, but the global error set still contains an entry for 0 which represents no error.

Reply:
Understood, thanks for clarifying jacobly.